base/access_provider/access_provider_test.go

```diff
@@ -0,0 +1,105 @@
+package access_provider
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"math/rand"
+	"os"
+	"strconv"
+	"testing"
+	"time"
+
+	"github.com/raito-io/cli/base/data_source"
+	"github.com/raito-io/cli/common/api/data_access"
+	"github.com/stretchr/testify/assert"
+)
+
+func init() {
+	rand.Seed(time.Now().UnixNano())
```

Comment:
Probably doesn't make sense to do this here as rand is a global library. Can we move this to somewhere global (e.g. main or root command)?

Reply:
Didn't see this is a test file. Closing this comment.